lib/grovepi/pivotpi.ex

```diff
@@ -0,0 +1,62 @@
+defmodule GrovePi.PivotPi do
+  @moduledoc """
+  This module lets you interact with the
+  [PivotPi](https://www.dexterindustries.com/pivotpi-tutorials-documentation/)
+  through the [GrovePi](https://www.dexterindustries.com/grovepi/).
+
+  Plug your PivotPi into the GrovePi I2C-1 port.  Plug your servo motor into
+  channel 1. You must initialize the PivotPi board using `PivotPi.start()`.
+
+  ```elixir
+  iex> PivotPi.start()
+  :ok
+  iex> PivotPi.angle(1, 180)
+  :ok
+  iex> PivotPi.angle(1, 90)
+  :ok
+  iex> PivotPi.angle(1, 0)
+  :ok
+  iex> PivotPi.led(1, 100)
+  :ok
+  iex> PivotPi.led(1, 50)
+  :ok
+  iex> PivotPi.led(1, 0)
+  :ok
+  ```
+  """
+
+  alias GrovePi.PivotPi.PCA9685
+
+  @doc """
+  Move the Servo motor to a new position.  Accepts angle from 0-180.
+  """
+  def angle(channel, angle) do
+    pwm_to_send = 4095 - translate_to_pwm(angle)
+    device_channel = channel - 1
+    PCA9685.set_pwm(device_channel, 0, pwm_to_send)
+  end
+
+  defp translate_to_pwm(angle_value) do
+    value_scaled = angle_value / 180
+    round(150 + (value_scaled * 450))
+  end
+
+  @doc """
+  Control the PivotPi LEDs.  Channel # should match Servo #.  Accepts % from 0-100.
+  """
+  def led(channel, percent) do
+    pwm_to_send = round(percent * 40.95)
```

Comment:
Is the `40.95` related to the `4095` above?

Reply:
For a 12-bit value the maximum representable value is 4095.  Since percent is represented as a whole number, the calculation is:

`(percent / 100) * 4095` 

I'll update the calculation to be more explicit.